dashboard/src/shared/AppRepository.ts

```diff
@@ -17,12 +17,9 @@ export class AppRepository {
   }
 
   public static async resync(cluster: string, name: string, namespace: string) {
-    const repo = await AppRepository.get(cluster, name, namespace);
-    repo.spec.resyncRequests = repo.spec.resyncRequests || 0;
-    repo.spec.resyncRequests++;
-    const { data } = await axiosWithAuth.put(
-      AppRepository.getSelfLink(cluster, namespace, name),
-      repo,
+    const { data } = await axiosWithAuth.post(
```

Comment:
Yes, technically it is perfectly feasible. However, webhooks (almost) always use `POST` method for sending requests mainly for: I) semantically, "refresh" is an action and it is something you want to get executed, not simply retrieve a value; II) it will trigger a subsequent operation that may not be idempotent.

At the moment, this POST method will ignore any payload sent in the body, that's the reason for sending null at the frontend.

Reply:
:thinking: okay, makes sense